data-reconciliation-app/docs/adr/02-data-schema-strategy.md

```diff
@@ -6,15 +6,15 @@ Decided
 
 ## Context
 
-We need to build an API that ingests member data into our system. Once ingested, we also need to store this data in the K-V store. Members will decide on a data schema which will be used for ingesting, data reconciliation and reporting.
+We need to build an API that ingest members data into our system. Once ingested, we need to store this data in the K-V store. Members will decide on a data schema that will be used for ingesting, data reconciliation, and reporting.
 
 There are multiple strategies for data schema definition and usage. In this document, we lay out our decision and some other alternatives considered.
 
 ## Alternatives
 
 ### Option1: Fully Flexible Data Schema
 
-In this strategy, members define their data schema and submit it as part of data ingestion. The API will take the schema and map it to an internal universal schema before storing in the K-V store. When a member requests a data reconciliation report, the API will generate the report using the internal universal schema. This will result in member to understand the schema of the report and possibly convert it back to their custom format to use at their side.
+In this strategy, members define their data schema and submit it as part of data ingestion. The API will take the schema and map it to an internal universal schema before storing it in the K-V store. When a member requests a data reconciliation report, the API will generate the report using the internal universal schema. This will result in members understanding the schema of the report and possibly converting it back to their own custom format to use it at their side.
```

Comment:
```suggestion
In this strategy, members define their data schema and submit it as part of data ingestion. The API will take the schema and map it to an internal universal schema before storing it in the K-V store. When a member requests a data reconciliation report, the API will generate the report using the internal universal schema. This will result in members understanding the schema of the report and possibly converting it back to their own custom format to use on their side.
```